src/modules/qt/filter_gpstext.cpp

```diff
@@ -252,7 +254,31 @@ static void gps_point_to_output(mlt_filter filter,
             snprintf(gps_text, 4, "%s", bearing_to_compass(raw.bearing));
         } else
             snprintf(gps_text, 4, "%s", bearing_to_compass(crt_point.bearing));
-    } else if (!strncmp(keyword, "gps_vdist_up", strlen("gps_vdist_up"))
+    }
+	else if (!strncmp(keyword, "gps_cadence", strlen("gps_cadence")) && crt_point.cad!=GPS_UNINIT) {
+		if (strstr(keyword, "RAW")) {
+			if (raw.cad == GPS_UNINIT) return;
+			snprintf(gps_text, 10, "%.0f", raw.cad);
+		}
+		else
+			snprintf(gps_text, 10, "%.0f", crt_point.cad);
```

Comment:
One thing clang-format does not enforce is matching usage of curly braces. I prefer that if the `if` block has curly braces so should each other block such as `else` here.